cmd/habitat/community/handlers.go

```diff
@@ -0,0 +1,21 @@
+package community
```

Comment:
so in my PR we have community as an "app" but I think since it's so core (the basic functinos of adding a member, joining, removing etc.) should be in habitat. and if we want to support different "types" of communities down the road (like perhaps setting an admin for some and not others) that can be at a higher level. So probably in a future PR I will move the logic from the handlers I created in the app into here. lmk what you think

Reply:
Yeah that sounds like a good way to merge our branches